schemas/database/database.md

```diff
@@ -0,0 +1,36 @@
+# Databases client spans
+
+Calls to databases should be tracked as client spans.
+
+Span `name` should be set to low cardinality value representing the statement
+executed on database. It may be stored procedure name (without argument), sql
+statement without variable arguments, etc. When it's impossible to get any
+meaningful representation of the span `name`, it can be populated using the same
+value as `db.instance`.
+
+**TODO: we might need have separate specs for other database types like Redis
+and such.**
+
+Note, Redis, Cassandra, HBase and other storage systems may reuse the same
+attribute names.
+
+**TODO: Agree to use `type` instead of `component`?**
+
+| Attribute name | Notes and examples |
+|:---------------|:-------------------|
+| `type`         | Database driver name or database name (when known) `JDBI`, `jdbc`, `odbc`, `postgreSQL`. |
+| `db.type`      | Database type. For any SQL database, `"sql"`. For others, the lower-case database category, e.g. `"cassandra"`, `"hbase"`, or `"redis"`. |
+| `db.instance`  | Database instance name. E.g., In java, if the jdbc.url=`"jdbc:mysql://127.0.0.1:3306/customers"`, the instance name is `"customers"`. |
+| `db.statement` | A database statement for the given database type. E.g., for `db.type="sql"`, `"SELECT * FROM wuser_table"`; for `db.type="redis"`, `"SET mykey 'WuValue'"`. |
+| `db.user`      | Username for accessing database. E.g., `"readonly_user"` or `"reporting_user"` |
+
+For database client calls, peer information SHOULD be collected.
+
+| Attribute name  | Notes and examples |
```

Comment:
Maybe peer should be a standalone document, because you may duplicate this a lot.

Reply:
I also think this could as well be a standalone document.

Reply:
there will be standalone document. I believe we will need to highligh some specifics for databases. I can do this as separate PR